meson.build

```diff
@@ -3538,12 +3528,10 @@ status = [
                                                               get_option('debug-tty')),
         'TTY GID:                           @0@'.format(tty_gid),
         'users GID:                         @0@'.format(substs.get('USERS_GID')),
-        'maximum system UID:                @0@'.format(system_uid_max),
-        'maximum system GID:                @0@'.format(system_gid_max),
-        'minimum dynamic UID:               @0@'.format(dynamic_uid_min),
-        'maximum dynamic UID:               @0@'.format(dynamic_uid_max),
-        'minimum container UID base:        @0@'.format(container_uid_base_min),
-        'maximum container UID base:        @0@'.format(container_uid_base_max),
+        'system UIDs:                       @0@–@1@'.format(conf.get('SYSTEM_UID_MIN'), conf.get('SYSTEM_UID_MAX')),
```

Comment:
I thought we only use .. because it's hard to type n-dash on a keyboard?

Reply:
I use … mostly because it's so pretty. (and kinda how mathematicians write ranges, no?)

(i mean, ignore me on this, it really doesn't matter at all...)